Ex1.java

```diff
@@ -0,0 +1,134 @@
+import java.util.HashMap;
+
+public class Ex1 {
+
+    /**
+     * The function receives two strings and checks if one is a permutation
+     * of the other
+     * @param firstStr

```

Comment:
Such empty documentation comments are not really nice and quite often it is better to just remove them.